crypto/hpke/hpke_util.c

```diff
@@ -82,18 +130,26 @@ int ossl_hpke_kdf_expand(EVP_KDF_CTX *kctx,
 int ossl_hpke_labeled_extract(EVP_KDF_CTX *kctx,
                               unsigned char *prk, size_t prklen,
                               const unsigned char *salt, size_t saltlen,
+                              const char *protocol_label,
                               const unsigned char *suiteid, size_t suiteidlen,
                               const char *label,
                               const unsigned char *ikm, size_t ikmlen)
 {
     int ret = 0;
     size_t labeled_ikmlen = 0;
-    unsigned char labeled_ikm[LABELED_EXTRACT_SIZE];
+    unsigned char *labeled_ikm = NULL;
     WPACKET pkt;
 
+    labeled_ikmlen = strlen(LABEL_HPKEV1) + strlen(protocol_label)
+        + suiteidlen + strlen(label) + ikmlen;
```

Comment:
maybe indent this to the stlren of the first lien

Reply:
forget if it was that or not but the check-format script complained about something like that - will play about see if that was it

Reply:
yeah that seem to be a quirk of the check-format script - it prefers line 144 indented only 8 octets for some reason